src/main/java/org/testeditor/fixture/host/net/Connection.java

```diff
@@ -199,37 +203,39 @@ public void run() {
      * A s3270 command will be executed with this method. The whole
      * communication with s3270 will be accessed through this method.
      */
-    public Result doCommand(final String command) {
+    public Result doCommand(final String commandString, Command command) {
```

Comment:
what do I need the commandString for, isn't this String part of the Command object itself?

Reply:
We discussed that and a new ticket is created for wrapping the X3270 driver.